free-3.md

```diff
@@ -10,34 +10,43 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This DSL would help anyone manage their time better. As of right now, the current means of time planning is a static calendar. However, this has a lot of disadvantages as it can be hard to tell which plans are flexible (like homework) and which are not. By using a DSL to add and remove events from a calandar, the computer can explore options, detect user patterns, and help make suggestions for when to schedule / reschedule events.
```

Comment:
This actually reminds me of if this then that from one of the 'is-this-a-dsl' games. You might want to draw some inspiration from it. I think this domain has a lot of room for out-of-the-box thinking.